pkg/apis/kops/validation/validation.go

```diff
@@ -1304,6 +1308,22 @@ func validateNodeLocalDNS(spec *kops.ClusterSpec, fldpath *field.Path) field.Err
 	return allErrs
 }
 
+func validateCertManager(spec *kops.CertManagerConfig, fldPath *field.Path) (allErrs field.ErrorList) {
+	allErrs = field.ErrorList{}
+
+	if fi.BoolValue(spec.Enabled) && fi.BoolValue(spec.External) {
+		allErrs = append(allErrs, field.Forbidden(fldPath, "Cert manager must be either enabled or external but not both"))
+	}
+	if fi.BoolValue(spec.External) && fi.StringValue(spec.Image) != "" {
```

Comment:
This seems a bit overkill. I think we can just document that when `external` is set, any other flag is ignored.

Reply:
Removed the last 2 checks.